centreon/src/Centreon/Infrastructure/Monitoring/Timeline/TimelineRepositoryRDB.php

```diff
@@ -300,7 +300,7 @@ private function prepareQueryForTimelineStatusEvents(
             FROM `:dbstg`.`logs` l
             WHERE l.host_id = :host_id
             AND (l.service_id = " . ($serviceId !== null ? ':service_id)' : '0 OR l.service_id IS NULL)') . "
-            AND l.msg_type IN (0,1,8,9)
+            AND l.msg_type IN (0,1)
             AND l.output NOT LIKE 'INITIAL % STATE:%'
```

Comment:
```suggestion

```
I propose to also remove this line since the match is linked to msg_types 8 and 9.
Thanks.